.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Unit Tests
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        go: [1.13, 1.14]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
don't test on windows atm, `gofmt` chokes due to line ending issues on almost every file 😞